odoo/addons/base/models/res_users.py

```diff
@@ -474,6 +474,12 @@ def _check_action_id(self):
         action_open_website = self.env.ref('base.action_open_website', raise_if_not_found=False)
         if action_open_website and any(user.action_id.id == action_open_website.id for user in self):
             raise ValidationError(_('The "App Switcher" action cannot be selected as home action.'))
+        for user in self:
+            if user.action_id.type == "ir.actions.client":
+                action = self.env["ir.actions.client"].browse(user.action_id.id)  # magic
+                if action.tag == "reload":
+                    raise ValidationError(_('The "%s" action cannot be selected as home action.') % action.name)
```

Comment:
I think you can use the `_('string %s', parameter1)` syntax in 15.0

Reply:
Done and tested :+1: